.github/ISSUE_TEMPLATE/BUG_REPORT.yaml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["symfony/ux"]
+body:
+  - type: markdown
+    attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report!
+    - type: dropdown
+    id: package
+    attributes:
+      label: Package
+      description: What package is concerned by this $$
+      options:
+        - Autocomplete
+        - Live Component
+        - Icons
+        - Translator
+        - Twig Component
+        - Autre
```

Comment:
`Autre` -> `Other` :) 

Should we add all components?

```
        - Autocomplete
        - Chartjs
        - Cropperjs
        - Dropzone
        - Icons
        - LazyImage
        - LiveComponent
        - Notify
        - React
        - StimulusBundle
        - Svelte
        - Swup
        - TogglePassword
        - Translator
        - Turbo
        - TwigComponent
        - Typed
        - Vue
```

Reply:
I had the idea to select only the most frequent in support... as a big list is always something a bit frightening :|

Reply:
Hahah yes of course baguette... 

I have really no strong opinion, not even sure it's better than "free textarea'... i think we should try something and iterate (and i'm not against starting with all of them :) )

i'm waiting the "staff" to give their opinion (when they have time, this has a -5000 prioriy like no others before 😅 )